site/static/_redirects

```diff
@@ -9,7 +9,7 @@
 /docs/tasks/rbac                           /docs/tasks/manage/rbac                           301
 /docs/tasks/run_job_with_workload_priority /docs/tasks/manage/run_job_with_workload_priority 301
 /docs/tasks/setup_multikueue               /docs/tasks/manage/setup_multikueue               301
-/docs/tasks/setup_sequential_admission     /docs/tasks/manage/setup_sequential_admission     301
+/docs/tasks/setup_wait_for_pods_ready      /docs/tasks/manage/setup_wait_for_pods_ready      301
```

Comment:
```suggestion
/docs/tasks/setup_sequential_admission      /docs/tasks/manage/setup_wait_for_pods_ready      301
/docs/tasks/manage/setup_sequential_admission      /docs/tasks/manage/setup_wait_for_pods_ready      301
```

Reply:
these are redirect so that old links work

Reply:
Yeah, fixed now